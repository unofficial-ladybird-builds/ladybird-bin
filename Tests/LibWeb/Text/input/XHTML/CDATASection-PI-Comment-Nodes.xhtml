<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
<body>
<div id="elem"><![CDATA[beep]]><?test test?><!-- I am a comment --><p>Hello</p><p>World</p></div>
<script src="../include.js"></script>
<script>
    test(() => {
        const elem = document.getElementById("elem");
        println(`elem.childNodes.length: ${elem.childNodes.length}`);
        println(`child node 0: ${elem.childNodes[0].toString()}, ${elem.childNodes[0].nodeType}`);
        println(`child node 1 ${elem.childNodes[1].toString()}, ${elem.childNodes[1].nodeType}`);
        println(`child node 2: ${elem.childNodes[2].toString()}, ${elem.childNodes[2].nodeType}`);
    });
</script>
</body>
</html>
