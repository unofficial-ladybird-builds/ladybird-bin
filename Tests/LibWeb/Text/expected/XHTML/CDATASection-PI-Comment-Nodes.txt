elem.childNodes.length: 5
child node 0: [object CDATASection], 4
child node 1 [object ProcessingInstruction], 7
child node 2: [object Comment], 8
